### PR TITLE
Fix the build

### PR DESCRIPTION
**Pull request by mistercrunch**
The travis build has been failing for 2 reasons recently
* pylint takes > 10 minutes without outputing
* bad merge confict auto resolve in controls.jsx

### CODE REVIEW

**Review by timifasubaa — APPROVED** [review]

File: tox.ini [hunk 1]
```diff
@@ -71,12 +71,10 @@ commands =
...
     -rrequirements.txt
     -rrequirements-dev.txt
-    pylint
 
 [tox]
 envlist =
     eslint
     flake8
     javascript
-    pylint
```
Comment (john-bodley) [inline tox.ini]: Why was this removed? `pylint` is a valid environment, i.e., `tox -e pylint`.
